sumologic/provider.go

```diff
@@ -18,14 +18,19 @@ func Provider() terraform.ResourceProvider {
 		Schema: map[string]*schema.Schema{
 			"access_id": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("SUMOLOGIC_ACCESSID", nil),
 			},
 			"access_key": {
 				Type:        schema.TypeString,
-				Required:    true,
+				Optional:    true,
 				DefaultFunc: schema.EnvDefaultFunc("SUMOLOGIC_ACCESSKEY", nil),
 			},
+			"auth_jwt": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				DefaultFunc: schema.EnvDefaultFunc("SUMOLOGIC_AUTHJWT", nil),
+			},
```

Comment:
Since we're only using jwt auth internally at the moment, I'm not sure if we even want/need to add it to the schema. Maybe just having it set as env variable only (and retrieving it from `os.Getenv` in the code) is enough.

Reply:
+1

Reply:
Done.